week-3/Homework/mandatory/5-journey-planner.js

```diff
@@ -1,14 +1,129 @@
+/*
+ Before we go the big story; we will introduce more string methods.
+ Some of the methods you're using in Array have similar ones with strings.
+ Methods like : IndexOf, Include, Search, Slice , Spilt and more.
+ 
+ You can always google how a method of a string works!
+ Here are links to some of those:
+  - https://www.w3schools.com/js/js_string_methods.asp
+  - https://javascript.info/string#quotes
+ Now let's do this small exercise
+ 
+ Using string methods update the checkCodeIsThere() function
+  - The function will have a string as a paramter
+  - The function should check if the string has the word "code" exists in the string
+  - If it does exist, return the index of it, if not return "Not found"
+ Hint: search for string methods like Includes and IndexOf.
+*/
+
+function checkCodeIsThere(stringText) {
+  let magicWord = "code";
+  //edit code below
+  if (stringText.includes("code")) {
+    return stringText.indexOf("code");
```

Comment:
The aim of the `magicWord` variable is to reuse it instead of manually typing its value as many times as many places you refer to that. It also makes the code refactoring easier as you only have to change its value in one place.